requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
-ansible<2.10
+ansible-core<2.13
```

Comment:
Should we also add this version control to the spec so that ansible-core will not go beyond this version in the future?

Reply:
Not a bad idea, given that RHEL 8.6 has 2.12 but C8S (and 8.7) are on 2.13, but this will need to be updated periodically for each RHEL ansible-core release.